ibis/pandas/core.py

```diff
@@ -343,7 +394,7 @@ def execute_until_in_scope(
         **kwargs,
     )
     computed = post_execute_(op, result, timecontext=timecontext)
-    return {op: computed}
+    return set_scope_item(op, computed, timecontext)
```

Comment:
I think the first arg to set_scope_item should be `scope`

Reply:
refactored and changed arg accrodingly